packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -117,10 +117,15 @@ import { QueryUsageViewer } from './QueryUsageViewer.js';
 
 export const tryToFormatSql = (sql: string): string => {
   try {
-    const formattedSql = formatSQL(sql);
+    const formattedSql = formatSQL(sql, { language: 'mysql' });
```

Comment:
why do we need to this?

Reply:
current one isn't formatting SQL for all the scenarios. Using this is formatting SQL with subselect